### PR TITLE
Small fixes [for case of packet fragmentation]

### PR DESCRIPTION
**Pull request by baffo32**
These adjustments allow the tests to pass if #101 is merged or used.

Incidentally also fixes some initialization issues when communicating with a real live board.

### CODE REVIEW

File: openBCIBoard.js [hunk 1]
```diff
@@ -1597,6 +1597,8 @@ function OpenBCIFactory () {
           this.curParsingMode = k.OBCIParsingNormal;
           this.buffer = null;
           this.emit('ready');
+        } else {
```
Comment (aj-ptw) [inline openBCIBoard.js]: Good!

**Comment by baffo32** [comment]
Just fixes, not tests.

Sorry, I guess this should probably be on top of #101.  The two relevant fixes are still applicable without #101, they just don't change any test results without it.  I am still learning about latency timers etc to possibly rework #101 to emulate the chip better.

**Comment by aj-ptw** [comment]
I'm surprised this passed tests because fragmentation is not in master

**Comment by baffo32** [comment]
will rebase onto #100

**Pull request closed without merging**